index.js

```diff
@@ -123,8 +123,14 @@ Promise.resolve()
 
         if (input.includes(file)) recompile.push(file)
 
+        const dependants = depGraph
+          .dependantsOf(file)
+          .concat(getAncestorDirs(file).flatMap(depGraph.dependantsOf))
+          // deduplicate
+          .filter((value, index, array) => array.indexOf(value) === index)
```

Comment:
Just convert to a set to deduplicate.

Reply:
👍  I have moved the deduplication and used a `Set`